### DISCUSSION TITLE
[Incorrect behavior][ARCropolis version 3.1.1]: Palutena's Guidance has completely broken

### DISCUSSION THREAD

**Discussion by DShip91** [Raytwo/ARCropolis#301] [Issues]
### Have you read the disclaimer before proceeding?

- [X] I have read the disclaimer.

### System Firmware Version

14.1.0

### Atmosphere Version

1.3.1

### Smash Version

13.0.1

### ARCropolis Version

3.1.1

### Version channel

Stable

### SD card formatting

FAT32

### Can this incorrect behavior be consistently reproduced?

Yes

### Expected behavior

When doing a Smash Taunt with Pit on Palutena's Temple, it's supposed to trigger a bit of dialogue with text, voice-acting, and character portraits from Kid Icarus Uprising

### Context

When I try to initiate Palutena's Guidance, all that appears is an empty overlay with no text, dialogue, or images.

### Configuration

Region: English (US)
Logging level: Warning
Beta updates: On
Legacy mod discovery: On
Debug: Off
Log to file: Off
Check for update on boot: On

### Minimal modpack to observe the issue
[DHD Guidance fix.zip](https://github.com/Raytwo/ARCropolis/files/8459920/DHD.Guidance.fix.zip)

**Comment by Raytwo** [comment]
Hey there, the minimal modpack field is there for you to *provide* the files so we can replicate the issue (or find what you did wrong).

Anyways I don't think an actual bug is happening here, it sounds more like a malformed xmsbt file, so feel free to send a modpack just in case.

**Comment by DShip91** [comment]
> Hey there, the minimal modpack field is there for you to _provide_ the files so we can replicate the issue (or find what you did wrong).
> 
> Anyways I don't think an actual bug is happening here, it sounds more like a malformed xmsbt file, so feel free to send a modpack just in case.

I just updated my post to include the modpack but this was even happening when the changes were part of an entire .msbt file.

**ETA:** This is happening to every Palutena's Guidance conversation, not just Duck Hunt's.

**Comment by ThatNintendoNerd** [comment]
I am able to reproduce this on 3.1.2. Testing with an empty mods directory in both `sd:/ultimate/mods/` and `rom:/arc/` as well as removing all Skyline plugins except for ARCropolis causes the issue to persist.

**Comment by Raytwo** [comment]
> I am able to reproduce this on 3.1.2. Testing with an empty mods directory in both `sd:/ultimate/mods/` and `rom:/arc/` as well as removing all Skyline plugins except for ARCropolis causes the issue to persist.

Got it, might be a byproduct of unsharing, gotta have to ask @blu-dev on that one I’m afraid.

**Comment by DShip91** [comment]
I just wanted to provide an update: Snake's codec calls seem to work but all of the spoken dialogue is in Japanese for some reason.

**Comment by Raytwo** [comment]
Could you also check the Starfox interaction, please? I wonder if this is affected too.
I think a solution would be to blacklist these files from being unshared, but it'd require testing.

**Comment by DShip91** [comment]
The Star Fox smash taunts have the same issue as the Metal Gear ones: The voices are all in Japanese.

**Comment by Raytwo** [comment]
Got it, I'll convert this into a conversation and reopen a proper issue for it.

**Reply by TripleLJ** [reply]
I can confirm that specific entries in Star Fox's smash taunts are in Japanese and specific ones appear blank. For example, the one where you taunt as Wolf and has Leon angry at Slippy for comparing him to a frog is always in Japanese in my experience, but many of Wolf's other ones are just blank. 

I also can always get Snake's codec on Lucario in Japanese and it hasn't appeared blank.

**Comment by Raytwo** [comment]
Please follow this [issue](https://github.com/Raytwo/ARCropolis/issues/312) for updates on when this gets investigated/fixed.
